README.md

```diff
@@ -132,21 +132,32 @@ how to operate in advance use cases, use case by use case
 explanation of all available options/configurations that can be used
 
 # Architecture
-![Container architecture](/docs/assets/container-architecture.jpg "Container architecture")
+
+There is mainly 2 stack you can use 1. rest.bitcoin.com techstack 2. Chris Troutner JS tech stack
+
+### 1. rest.bitcoin.com tech stack
+
+![rest.bitcoin.com Tech Stack](/docs/assets/rest-bitcoin-com_techstack.jpg "rest.bitcoin.com Tech Stack")
```

Comment:
Note that GitHub markdown does not support figure captions.

Reply:
Oh ok 🙂